source/tutorial/enforce-keyfile-access-control-in-existing-sharded-cluster-no-downtime.txt

```diff
@@ -0,0 +1,297 @@
+==========================================================================
+Enable mandatory user authentication in a sharded cluster without downtime
+==========================================================================
```

Comment:
This was a suggestion from Andy - I'm guessing that 'mandatory' is a stronger implication. I don't object in removing it, but I don't think it takes away from anything either.